requirements.txt

```diff
@@ -1,3 +1,6 @@
+datasets==1.11.0
+jinja2==3.0.1
 tensorflow==2.5.0
 torch==1.9.0
```

Comment:
Should probably double check that these are indeed the package versions that the modeling/engineering WG is using.